clients/cli/cmd/push.go

```diff
@@ -34,5 +35,6 @@ func initPush() {
 	AddFlag(pushCmd, "bool", "wait", "w", "Wait for files to be processed", false)
 	AddFlag(pushCmd, "string", "branch", "b", "branch", false)
 	AddFlag(pushCmd, "bool", "use-local-branch-name", "", "push from the branch with the name of your currently checked out branch (git or mercurial)", false)
+	AddFlag(pushCmd, "string", "tag", "", "Pre tag the uploaded keys with a specific tag", false)
```

Comment:
```suggestion
	AddFlag(pushCmd, "string", "tag", "", "Tag uploaded keys", false)
```

Would be great if you could adapt the description like this